### ISSUE TITLE
Surfingkeys visual highlighting hints not visible and compressed above view port line whilst using Dark Reader default Filter

### ISSUE THREAD

**Issue by rople380** [brookhong/Surfingkeys#1343]
## Prelude
* Could you find answer in below documents?

    1. [FAQ](https://github.com/brookhong/Surfingkeys/wiki/FAQ) / [常见问题](https://github.com/brookhong/Surfingkeys/wiki/%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98)
    1. [README](https://github.com/brookhong/Surfingkeys/blob/master/README.md) / [读我](https://github.com/brookhong/Surfingkeys/blob/master/README_CN.md)

No.

* Have you searched your problem in [issues](https://github.com/brookhong/Surfingkeys/issues)?

Yes.

## Error details

SurfingKeys: 0.9.67

Browser: 78.0.2 (64-bit) | Ubuntu 19.10 x86_64

URL: * (All sites)

## Context

Unable to view hints when [DarkReader](https://darkreader.org/) is running for visual mode highlighting with default Filter (see attached images below).

Gets jumbled, seems to stop below the top view port line, unable to see below that line, things get compressed if scrolled down.

![Visual Hint Without Dark Reader](https://user-images.githubusercontent.com/55679162/97079908-49b2b300-1643-11eb-926a-8133b0606109.png)

![Visual Hint With Dark Reader Default Filter](https://user-images.githubusercontent.com/55679162/97079909-4ae3e000-1643-11eb-9fda-39b78e7732e5.png)

Tested with Dark Reader Filters: Filter (Default)

Filter+ is a black screen (maybe a GPU issue), filters Static/Dynamic work correctly.

**Comment by brookhong** [comment]
Can not reproduce this from my side.

![image](https://user-images.githubusercontent.com/288207/98261283-42829200-1fbf-11eb-96d0-aed226f7671c.png)

**Comment by rople380** [comment]
Can you give me details of your system/software/extensions?

That way I can replicate.

**Comment by brookhong** [comment]
SurfingKeys: 0.9.67
Chrome: 86.0.4240.183
OS:  Above screenshot is from Mac 10.15.7, and below is from Windows 10

![image](https://user-images.githubusercontent.com/288207/98366507-11619a80-206f-11eb-8dbd-d483a49ca3dd.png)

**Comment by rople380** [comment]
Yes I don't have the problem in Chrome. It's in FireFox that I have the issue.

**Comment by brookhong** [comment]
And this is on
SurfingKeys: 0.9.67
Firefox: 82.0.1
OS:  Mac 10.15.7

![image](https://user-images.githubusercontent.com/288207/98490973-4ab42900-226e-11eb-97a7-d765295bf233.png)

**Comment by rople380** [comment]
Upgraded FireFox: 82.0.2 (64-bit)

Issue remains, ran in private mode to avoid extension interaction, issue remains.

Perhaps its a driver rendering issue?

`sudo lshw -C video`

Produces:

```
  *-display
       description: VGA compatible controller
       product: UHD Graphics 630 (Desktop 9 Series)
       vendor: Intel Corporation
       physical id: 2
       bus info: pci@0000:00:02.0
       version: 02
       width: 64 bits
       clock: 33MHz
       capabilities: vga_controller bus_master cap_list rom
       configuration: driver=i915 latency=0
       resources: irq:126 memory:de000000-deffffff memory:c0000000-cfffffff ioport:f000(size=64) memory:c0000-dffff

```
